core/templates/dev/head/app.js

```diff
@@ -374,7 +374,6 @@ oppia.config(['toastrConfig', function(toastrConfig) {
     messageClass: 'toast-message',
     progressBar: false,
     tapToDismiss: true,
-    timeOut: 1500,
```

Comment:
A question -- the removed key is timeOut and the key that seems to be replacing it is timeout. Is that intentional?

Reply:
Oh! Sorry it's an error from my end, correcting the same. Thanks for pointing this out.

Reply:
But why wasn't it caught? Did you not test it manually?

Reply:
Done :+1: